bin/migrate-tips.py

```diff
@@ -3,14 +3,20 @@
 
 db = db(env())
 
-slugs = db.all("""
-    SELECT slug
-      FROM teams
-     WHERE is_approved IS TRUE
+teams = db.all("""
+    SELECT distinct ON (t.slug) t.*::teams
+      FROM teams t
+      JOIN tips ON t.owner = tips.tippee	-- Only fetch teams whose owner have tips.
+     WHERE t.is_approved IS TRUE           	-- Only fetch approved teams.
+       AND NOT EXISTS (                  	-- Make sure not already migrated.
+            SELECT 1
```

Comment:
Yep, confirmed. 

<img width="623" alt="screen shot 2016-01-24 at 3 31 37 pm" src="https://cloud.githubusercontent.com/assets/3893573/12535637/95ce4d0e-c2af-11e5-99bc-fedb5bdc1d96.png">

This is sublime text - See the difference? The `.`s are spaces, and `-`s are tabs